flight/PiOS/STM32F4xx/pios_video.c

```diff
@@ -95,21 +95,17 @@ volatile uint16_t active_line = 0;
 
 const struct pios_video_type_boundary *pios_video_type_boundary_act = &pios_video_type_boundary_pal;
 
+int8_t video_type_act = VIDEO_TYPE_NONE;
```

Comment:
Not a big fan of sharing a global between a low-level driver and high-level module.. :/